### DISCUSSION TITLE
Can't run/Can't Install/Not smart enough to check closed issues

### DISCUSSION THREAD

**Discussion by bhardwajbj** [D4rkS3c-Group/HiddenEyeReborn#63] [Q&A] [answered]

**Comment by Morsmalleo** [comment]
Pip install hiddeneye_reborn but it won't work, its still in development

**Comment by sTiKyt** [comment]
Opening this back as main "issue" for those spamming me with questions

New issues with same context will be referencing this one and people making those will be banned permanently.

**Comment by sTiKyt** [comment, marked as answer]
HiddenEye Reborn IS NOT READY FOR END USER, IT DOESN'T WORK YET, DON'T INSTALL IT.

**Reply by ELREYNINFA** [reply]
aa ok

**Reply by Doujinx** [reply]
when it will be ready ?

**Reply by sTiKyt** [reply]
> when it will be ready ?

No ETA, maybe it will be, maybe it won't

**Comment by Chels78** [comment]
Aa ok

On Wed, Dec 22, 2021 at 3:38 AM ELREYNINFA ***@***.***>
wrote:aa
